core/state/snapshot/snapshot.go

```diff
@@ -560,16 +560,6 @@ func diffToDisk(bottom *diffLayer) *diskLayer {
 			batch.Delete(key)
 			base.cache.Del(key[1:])
 			snapshotFlushStorageItemMeter.Mark(1)
-
-			// Ensure we don't delete too much data blindly (contract can be
-			// huge). It's ok to flush, the root will go missing in case of a
-			// crash and we'll detect and regenerate the snapshot.
-			if batch.ValueSize() > ethdb.IdealBatchSize {
```

Comment:
Can we keep the code for now and replace `ethdb.IdealBatchSize` with `64 * 1024 * 1024` in both invocations? When removing SELFDESTRUCT lands in Cancun, we can nuke it out according to the original PR. In between we'd at least have a more robust code.